portworx-mount-k8/step2.md

```diff
@@ -0,0 +1,3 @@
+Persistent Volume Claim
+
+>>Q3: Create a PVC called 'nginx-pvc' and make use of the storage class 'px-sc'. Storage Request: 1Gi, AccessModes: RWO<< 
```

Comment:
Might be worth a helpful hint to link to in references? https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims

Reply:
done.